Add Common CA fields

XolphinMartijn opened the pull request:
Please see comments below prior to merging
@srdavidson  ready for a review

XolphinMartijn committed:
Common CA Fields

XolphinMartijn commented:
Note: This draft adds a new requirement:
| __Field__   | __Minimum__ | __Maximum__ |
| --          | ----        | ----        |
| `notBefore` | One day prior to the time of signing | The time of signing |
| `notAfter`  | The time of signing | Unspecified |

Unless there's pushback, I suggest we keep this to align with the TBRs

XolphinMartijn commented:
Note: Section 7.1.2.10.5 contains deviations from the TBRs. Specifically, this allows for not just 1 but multiple Reserved Policy OIDs in policy restricted CAs

XolphinMartijn commented:
Note: Section 7.1.2.10.8 received some updates compared to the TBRs. Please perform a review before merging

XolphinMartijn committed:
More COmmon CA Fields

aww-aww commented on SBR.md:
```diff
@@ -2007,22 +2007,165 @@ Not applicable.
+| __Field__   | __Minimum__ | __Maximum__ |
+| --          | ----        | ----        |
+| `notBefore` | One day prior to the time of signing | The time of signing |
+| `notAfter`  | The time of signing | Unspecified |
```
s/Unspecified/The maximum allowed for the certificate type/

XolphinMartijn replied:
```suggestion
| `notAfter`  | The time of signing | The maximum allowed for the certificate type |
```

XolphinMartijn committed:
Update SBR.md

srdavidson committed:
Refactor SBR.md for consistency and clarity

Updated various sections for consistency and clarity, including capitalization and formatting changes for terms like 'Certificate' and 'Certificates'.

srdavidson committed:
Merge branch 'profiles' into common_ca_fields
